README.md

```diff
@@ -186,6 +186,42 @@ and removing them by logging out:
 $ deadline auth logout
 ```
 
+## AWS Credentials Integration
+
+You can use the Deadline Cloud client to obtain temporary AWS credentials for a queue and use them with the AWS CLI or SDK. This enables you to create AWS profiles that have queue-specific permissions for use in programmatic workflows.
+
+To export credentials for a queue in a format compatible with the AWS SDK credentials_process interface:
+
+```sh
+$ deadline queue export-credentials --farm-id farm-1234567890abcdefg --queue-id q-12345abcdef --mode USER
+{
+  "Version": 1,
+  "AccessKeyId": "ASIA...",
+  "SecretAccessKey": "wJalr...",
+  "SessionToken": "AQoD...",
+  "Expiration": "2025-04-08T20:00:46Z"
+}
+```
+
+You can then reference this command in your AWS config file to create a profile that uses these credentials:
+
+```
+[profile deadline-queue]
+credential_process = deadline queue export-credentials --farm-id farm-1234567890abcdefg --queue-id q-12345abcdef --mode USER
```

Comment:
Might be nice to show the --profile option in this call to explicitly connect where that's coming from.